From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Fedora Python maintainers <python-devel@lists.fedoraproject.org>
Date: Wed, 15 Jul 2020 15:23:56 +0200
Subject: [PATCH] 00137-skip-distutils-tests-that-fail-in-rpmbuild.patch

00137 #
Some tests within distutils fail when run in an rpmbuild:
---
 Lib/distutils/tests/test_bdist_rpm.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Lib/distutils/tests/test_bdist_rpm.py b/Lib/distutils/tests/test_bdist_rpm.py
index 8248c08b19b..78ed410c79d 100644
--- a/Lib/distutils/tests/test_bdist_rpm.py
+++ b/Lib/distutils/tests/test_bdist_rpm.py
@@ -29,6 +29,7 @@ setup(name='foo', version='0.1', py_modules=['foo'],
 
 """
 
+@unittest._skipInRpmBuild("don't try to nest one rpm build inside another rpm build")
 class BuildRpmTestCase(support.TempdirManager,
                        support.EnvironGuard,
                        support.LoggingSilencer,
